src/mixins/canvas_events.mixin.js

```diff
@@ -715,7 +721,7 @@
 
       if (target) {
         var alreadySelected = target === this._activeObject;
-        if (target.selectable) {
+        if (target.selectable && target.activeOn !== 'up') {
```

Comment:
can we specify === 'down' instead of 'up'?

Reply:
How do we do for json dump on this? We often do not export events/interactions property. But i feel like this is different. I do not want to resolve this in this single PR. Just give it a thought

Reply:
sure, my thinking in doing it this way was that invalid values would just revert the behavior back to the default but I can change it if you don't think that's necessary

Reply:
my intention was to exclude this prop from the json since I feel it's not likely to be changed much and can always be manually included in the `extraProperties` array